### ISSUE TITLE
Prevent printing an empty line in chat if broadcast.get("message") returns an empty string.

### ISSUE THREAD

**Issue by Owehttamy** [WarzoneMC/Warzone#791]
When working on the map "Not So Spleef", I ran into a dilemma where I wanted to run commands without relying on command blocks and having to hide them within the map. So, I tried out the broadcast option in the time module. I found that if you use broadcasts within the time module in a map's JSON file to execute commands, but leave the message attribute with an empty string, the module still prints an empty line.

I'd like to request a feature that either adds an exception that checks if the message string is empty to prevent blank lines from spamming within chat, or just remove the requirement of including the message attribute altogether, which is likely more resource-efficient.

This would make many maps much more intuitive and would provide much more ease to tweak and update, due to all special functionality in maps being within a JSON file, rather than the map's region files, which would need to be pruned every time a change is required.

In short, when you try to execute commands with the broadcast module without writing a message, an empty line is sent in chat when the broadcast is run. My request is to remove the unnecessary empty lines so they're not excessively creating gaps in the chat, and instead the command is run silently in the background.
